tests/tests.rs

```diff
@@ -200,35 +200,29 @@ fn empty() {
 #[test]
 #[cfg(feature = "backtrace")]
 fn has_backtrace_depending_on_env() {
-    use std::env;
+    use std::process::Command;
+    use std::path::Path;
 
-    error_chain! {
-        types {}
-        links {}
-        foreign_links {}
-        errors {
-            MyError
-        }
-    }
-
-    let original_value = env::var_os("RUST_BACKTRACE");
+    let cmd_path = if cfg!(windows) {
+        Path::new("./target/debug/has_backtrace.exe")
+    } else {
+        Path::new("./target/debug/has_backtrace")
```

Comment:
I would use `cargo run --bin` and maybe put it somewhere `cargo` wouldn't pick up by default. I can try that if it sounds good.